client/src/App.js

```diff
@@ -55,14 +56,15 @@ class App extends React.Component {
                             <Route path="/map" component={MapPage}/>
                             <Route exact path="/" component={HomePage}/>
                             <Route path="/login" component={LoginPage}/>
-                            <Route path="/signup" component={SignupPage}/>
+                            <Route exact path="/signup" component={SignupPage}/>
```

Comment:
What does `exact` do here?

Reply:
without it `/signup/confirmation` would map to it, but I just want `/signup` to.